userChrome.css

```diff
@@ -46,3 +46,41 @@
 #reload-button {
 	margin-block-start: -2px !important;
 }
+
+
+/* context menuitem padding and rounded hover style changes */
```

Comment:
Like I mentioned in the previous PR, this patch must be tested in Linux before I can merge it because currently it breaks the context menu on this platform.

This should help you target windows 10 only:
`@media (-moz-os-version: windows-win10) { ... }`

Reply:
Ok will add it as well